crates/rrg/src/args.rs

```diff
@@ -75,9 +85,58 @@ fn parse_duration(value: &str) -> Result<Duration, String> {
     humantime::parse_duration(value).map_err(|error| error.to_string())
 }
 
+/// Decodes a Vector of hex digits to a Vector of byte values.
+fn hex_decode(hex: Vec<u8>) -> Result<Vec<u8>, HexDecodeError> {
```

Comment:
Accepting an owned vector that is consumed by the function is indication that you should probably accept a slice (`&[u8]`).

The original [`hex::decode`](https://docs.rs/hex/latest/hex/fn.decode.html) goes even a step further an accepts `AsRef<[u8]>`, so will work both with `Vec`s and slices without any additional borrows, although I don't think we need that level of generality here.

Reply:
Done.